CHANGELOG.rst

```diff
@@ -3,6 +3,24 @@ Release Notes
 
 Changes are ordered reverse-chronologically.
 
+
+0.10.0
+------
+
+ - Support for coach-specific content
+ - Content import/export is more reliable and easier to use
+ - Search has improved results and handles duplicate items
+ - Display of answer history in learner exercises is improved
+ - Login page is more responsive
+ - Windows-specific improvements and bug fixes
+ - New Kolibri configuration file
+ - Overall improved performance
+ - Auto-play videos
+ - Command to migrate content directory location
+
```

Comment:
No mention of the pdf renderer improvements :broken_heart:

Reply:
on it!

Reply:
remember to use this!
![image](https://user-images.githubusercontent.com/2367265/40992184-ed2dd22e-68aa-11e8-9cd2-449446d4cd0a.png)

Reply:
updated